MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflMainView.h

```diff
@@ -4,7 +4,7 @@
 #include "MantidKernel/System.h"
 #include "MantidQtAPI/AlgorithmRunner.h"
 #include "MantidQtCustomInterfaces/Reflectometry/IReflPresenter.h"
-#include "MantidQtCustomInterfaces/Reflectometry/ReflCommandBase.h"
+#include "MantidQtCustomInterfaces/Reflectometry/ReflCommand.h"
```

Comment:
You could actually avoid the include here and foward declare a lot of this instead. Unfortunately that would mean that you couldn't use your `typedefs`, but it would help decouple your compilation units.